### DISCUSSION TITLE
macos + Cmd-H

### DISCUSSION THREAD

**Discussion by pafnuteus** [elfmz/far2l#1359] [Q&A] [answered]
Подскажите, есть ли возможность в macos версии (у меня сейчас стоит 2.4.0-beta Darwin x86_64) отключить реагирование на комбинацию клавиш Cmd-H. В macos этот хоткей скрывает текущее активное окно, а фар его перехватывает и не скрывается.
Как я понял, far2l воспринимает клавишу command как ctrl, поскольку при этом срабатывает команда показать/скрыть hidden files.

**Comment by DEF29** [comment]
У меня такая же ситуация. Пробовал найти решение, но безуспешно.
3.2.57(1)-release (x86_64-apple-darwin18)

**Comment by elfmz** [comment]
Самое интересное что far2l ничего не предпринимает для такого перехвата. Скорее всего это сочетание обрабатывается не на уровне системы, а добровольно самим приложением. У far2l такой обработки нет, хотя, возможно, стоит добавить.
А можно список таких сочетаний для не-маковода, чтоб оценить масштаб проблемы и возможные конфликты с внутренними хоткеями?

**Reply by elfmz** [reply]
.. добавил обработку Cmd+H и Cmd+Q в текущем мастере

**Reply by pafnuteus** [reply]
мне лично только cmd-h надо) даже cmd-q как-то не приучился использовать.
спасибо за быстрое реагирование!
а с какой периодичностью выходят бинарные дистрибутивы? прошлый раз пытался вручную собирать, и не хватило таланта :)

**Reply by elfmz** [reply]
ну, периодичность тут примерно как у рандома..

**Reply by elfmz** [reply]
вот, вышел..

**Comment by pafnuteus** [comment, marked as answer]
офигеть, работает like a charm! спасибо спасибо спасибо! куда можно донат занести?

**Comment by DEF29** [comment]
Подтверждаю! У меня тоже все стало прекрасно на моей Mojave
Автору огромное спасибо за монументальные труды и наш любимый фар!!!
